### ISSUE TITLE
Featured Properties Custom Post Type

### ISSUE THREAD

**Issue by bhubbard** [idxbroker/wordpress-plugin#33]
Introduce Featured Properties as a Custom Post Type. The featured listing cpt would be public, but read only. I suggest the post type be named `idxbroker-featured`.

**Comment by amcn84** [comment]
Featured properties is still a system link. How would this benefit all clients?

**Comment by bhubbard** [comment]
I presented a demo at the IDX Broker Developer conference, showing how I thought it should work. I am not referencing the systemlink to featured properties. 

What I am suggesting is the WordPress plugin pulls all the data for featured properties thru an API call and stores them in WordPress as a custom post type. 

Once this is done, many features open up:
* WordPress users can view a listing of IDX Broker Featured Properties from the WordPress Dashboard
* Featured Properties can be searched and found via the default WordPress search box
* Widgets can be created to pull from the CPT instead of doing direct API calls
* Standard WordPress functions can be used to create custom templates based on IDX Featured Properties
* etc

Here is a link to a video I created and sent to IDX Broker:
https://dl.dropboxusercontent.com/u/782569/idxbrokerplugin-demo.mp4
